bots/tests-scan

```diff
@@ -143,6 +144,12 @@ def prioritize(status, title, labels, priority, context):
     elif state in [ "pending" ]:
         update = None
 
+    # Don't start working on "bot" pull requests automatically.
+    # The bot triggers them explicitly.
+    elif "bot" in labels():
+        priority = None
+        update = None
+
```

Comment:
This reverts the changes from #6845. If you would like to do this, it would make sense to do as a separate pull request with separate discussion.